tests/integration/tier0/non-destructive/basic-sanity-checks/main.fmf

```diff
@@ -61,10 +61,6 @@ tag+:
 
 
 /convert2rhel_version:
-    adjust+:
-        result: xfail
-        when: distro == alma-9, oracle-9, rocky-9, stream-9
-        because: We're unable to validate the version of convert2rhel, as there is no build GA for EL9 yet
```

Comment:
```
[2024-10-22T16:21:40+0000] DEBUG - Got a line without the C2R identifier: Convert2RHEL for OS 9                           2.7 kB/s | 512  B     00:00    
[2024-10-22T16:21:40+0000] DEBUG - Found 0 convert2rhel package(s)
[2024-10-22T16:21:40+0000] DEBUG - Found 0.00 to be latest available version
```

I guess this will still fail until the official rhel9 package of c2r is released. This should happen once the errata goes live. Am I right @danmyway ?

Reply:
@hosekadam I had a discussion with @danmyway about this and it seems that this can actually be removed now. Can you please try to remove this adjust: xfail part again. Sorry for doing this again :D